src/main/scala/com/amazon/deequ/analyzers/Distance.scala

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ * use this file except in compliance with the License. A copy of the License
+ * is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+package com.amazon.deequ.analyzers
+
+case class CategoricalBucket(value: String, count: Long)
+
+case class Categorical(buckets: List[CategoricalBucket])
+
+class Distance {
+
+    /** Calculate distance of numerical profiles based on KLL Sketch
+     *  This formula is based on  “Two-sample Kolmogorov–Smirnov test"
+     *  Reference: https://en.m.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test */
```

Comment:
Can you explain how this is related to a KS test?

Reply:
Absolutely. The formula of linf_robust is based on the “Two-sample Kolmogorov–Smirnov test", which is used to test whether two underlying one-dimensional probability distributions differ. Check out the “Two-sample Kolmogorov–Smirnov test" section of the reference link and you could find out where the item 1.8 * Math.sqrt((n + m) / (n * m)) in the formula is coming from (when alpha = 0.001, c(alpha) is approximate to 1.8).